src/test/cli.build.test.ts

```diff
@@ -16,7 +16,7 @@ const pkg = require('../../package.json');
 describe('Dev Containers CLI', function () {
 	this.timeout('120s');
 
-	const tmp = path.relative(process.cwd(), path.join(__dirname, 'tmp'));
+	const tmp = path.join(__dirname, 'tmp');
```

Comment:
Why is this needed?

Reply:
This is just how the tests call the locally built version of the package, but it was not doing well to handle changing the cwd of the spawned instances (since then the path would be different). So this changes it to be an absolute path instead of a relative path.